core/db/migrate/20220117100333_add_make_active_at_to_spree_products.rb

```diff
@@ -0,0 +1,8 @@
+class AddMakeActiveAtToSpreeProducts < ActiveRecord::Migration[5.2]
+  def change
+    add_column :spree_products, :make_active_at, :datetime
+    add_index :spree_products, :make_active_at
+    Spree::Product.where('available_on <= ?', Time.current).where(status: 'draft').update_all(status: 'active', updated_at: Time.current)
```

Comment:
@kgorazd we need to check if these products aren't discontinued, it's possible to have both available on and discontinue on date set at the same time, I think we should use the same SQL query as the old `available` scope